### PR TITLE
Push package to PyPi

### LINKED ISSUE #146: Automatically publish Python package on PyPi on release
Currently there is a package that has been manually released, but it is not automated.

https://pypi.org/project/cog/

We should also set some expectation around what this package is. Just the Python package doesn't actually do very much.

### PR DESCRIPTION
**Pull request by bfirsh**
Push the exact thing we install inside the container to PyPi.

This is not well documented – it's not clear how to use the PyPi package standalone. In short, you can run `python -m cog.server.http`, so long as there's a `cog.yaml` with a `predictor` key in it. Not ideal, but we can fix this up later.

I have tested this little script locally, but not tested the actual push to PyPi obviously. We can fix it live if it doesn't work. I have set the `PYPI_TOKEN` secret.

Closes #146

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -121,3 +121,17 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build Python package
+        run: |
+          # clean package built for go client
+          rm -rf python/dist
+          # set version
+          sed -i -E "s/0.0.1/${GITHUB_REF_NAME:1}/" setup.py
```
Comment (zeke) [inline .github/workflows/ci.yaml:129]: What does the `:1` expansion do here?

**Review by zeke — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (bfirsh) [reply .github/workflows/ci.yaml:129]: Removes first character. The tag is `v1.2.3`.

**Comment by bfirsh** [comment]
https://pypi.org/project/cog/

**Pull request merged**